src/graphql/subscription/map_async_iterator.py

```diff
@@ -43,9 +43,18 @@ async def __anext__(self) -> Any:
             aclose = ensure_future(self._close_event.wait())
             anext = ensure_future(self.iterator.__anext__())
 
-            pending: Set[Future] = (
-                await wait([aclose, anext], return_when=FIRST_COMPLETED)
-            )[1]
+            # Suppress the StopAsyncIteration exception warning when the iterator is cancelled.
```

Comment:
We use black and have a limit of 88 chars per line. You can check with "tox -e flake8" and "tox -e black".